app/views/foreman_leapp/job_templates/leapp_install.erb

```diff
@@ -0,0 +1,35 @@
+<%#
+kind: job_template
+name: Install Leapp
+job_category: Leapp - Install
+description_format: 'Install Leapp CI tool'
+provider_type: Ansible
+model: JobTemplate
+oses:
+- RedHat
+%>
+<%
+  os_major = @host.operatingsystem.major.to_i
+  rhel_compatible = @host.operatingsystem.family == 'Redhat' && @host.operatingsystem.name == 'RedHat'
+  repos = os_major == 7 ? ['rhel-7-server-rpms', 'rhel-7-server-extras-rpms'] :
+                          ['rhel-8-for-x86_64-baseos-rpms', 'rhel-8-for-x86_64-appstream-rpms']
```

Comment:
What would happen if I run this thing against a rhel9 machine?

Reply:
Then it will fail, because leapp is not for RHEL 9 now. When there is will be RHEL10 we'll have to update playbook anyway so then we can use case statement

Reply:
In that case, shouldn't it fail explicitly instead of letting it die while trying to enable rhel8 repos on a rhel9 machine?